fairseq/model_parallel/models/roberta/model.py

```diff
@@ -261,7 +261,7 @@ def base_architecture(args):
     args.pooler_dropout = getattr(args, 'pooler_dropout', 0.0)
     args.encoder_layers_to_keep = getattr(args, 'encoder_layers_to_keep', None)
     args.encoder_layerdrop = getattr(args, 'encoder_layerdrop', 0.0)
-    args.checkpoint_transformer_block = getattr(args, 'checkpoint_transformer_block', False)
```

Comment:
This was done in my previous commit:
https://github.com/SciPioneer/fairseq/commit/9627bd8a7493384b7e5b382e742bfee931a30c91